src/transformers/tokenization_utils_base.py

```diff
@@ -798,12 +798,13 @@ def as_tensor(value, dtype=None):
 
         return self
 
-    def to(self, device: Union[str, "torch.device"]) -> "BatchEncoding":
+    def to(self, device: Union[str, "torch.device"], non_blocking: bool = False) -> "BatchEncoding":
```

Comment:
To force passing `non_blocking` as a keyword argument, so later we can introduce additional positional or keyword args without breaking BC, what do you think?

```suggestion
    def to(self, device: Union[str, "torch.device"], *, non_blocking: bool = False) -> "BatchEncoding":
```